code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -74,6 +75,32 @@
 /obj/item/weapon/cautery/is_hot()
 	return 1
 
+/obj/item/weapon/cautery/laser //Not obtainable normally. Used for the laser scalpel parent.
```

Comment:
Why does this need to exist? Why not just make this the tier 1 cautery?

Reply:
It doesn't really need to. It's there so it can use an orange cautery for the orange laser scalpel. Both are technically unused in game but it does have a speed of 0.5 to make it fit where it would be used.
![cauteries](https://user-images.githubusercontent.com/28679186/31175511-7af1a8d0-a8dd-11e7-8d1e-65fd469c79f1.PNG)

Reply:
yes, the basic laser scalpel and mid laser scalpel's colors are swapped. This is intended.